### PR TITLE
Release 3.3.2 (#4596)

### PR DESCRIPTION
**Pull request by lucacome**
(cherry picked from commit d239c207534aaa4fc524fb276f8d44330dbe8571)

### CODE REVIEW

**Review by shaun-nx — APPROVED** [review]

**Review by jjngx — APPROVED** [review]

File: docs/content/app-protect-dos/installation.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ This document provides an overview of the steps required to use NGINX App Protec
 2. Clone the Ingress Controller repo:
 
     ```
```
Comment (Jcahilltorre) [inline docs/content/app-protect-dos/installation.md:22]:
```suggestion
    ```shell
```
Code blocks should to have a language defined (even if it's `none`). For shell instructions use `shell` or `bash` (please note that `console` is not recognized by hugo and breaks the highlighting)

Comment (Jcahilltorre) [reply docs/content/app-protect-dos/installation.md:22]: we might need to open a ticket to review all the code blocks in the docs and add the missing languages cc @ADubhlaoich

File: docs/content/app-protect-waf/configuration.md [hunk 2]
```diff
@@ -403,31 +403,31 @@ In this example we deploy NGINX Ingress Controller with NGINX Plus and NGINX App
 Create the application deployment and service:
 
   ```console
```
Comment (Jcahilltorre) [inline docs/content/app-protect-waf/configuration.md:405]:
```suggestion
  ```shell
```

Comment (Jcahilltorre) [reply docs/content/app-protect-waf/configuration.md:405]: console:
image

shell:
image


etc

**Pull request merged**